src/components/Tasks/TaskStatus.js

```diff
@@ -11,10 +44,10 @@ function TaskStatus() {
         margin-left: ${GU * 0.7}px;
       `}
       >
-        Open: Due today
+        {`Open: ${getDueDayText(dueDate)}`}
```

Comment:
I think we should be able to do this:

```suggestion
        Open: {getDueDayText(dueDate)}
```